crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -134,18 +134,22 @@ pub struct BuildSpecCommand {
 	/// [default: ./chain-spec.json].
 	#[arg(short = 'o', long = "output")]
 	pub(crate) output_file: Option<PathBuf>,
-	/// For production, always build in release mode to exclude debug features.
+	/// [DEPRECATED] For production, always build in release mode to exclude debug features.
```

Comment:
Why is this deprecated? There is nothing in the PR description and no comment here to advise why this is being done.